lizmap/modules/lizmap/lib/Commands/WMTSCacheClean.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Lizmap\Commands;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class WMTSCacheClean extends \Jelix\Scripts\ModuleCommandAbstract
+{
+    protected function configure()
+    {
+        $this
+            ->setName('wmts:cache:clean')
+            ->setDescription('Get seeding capabilities')
+            ->setHelp('')
+            ->addArgument('repository', InputArgument::REQUIRED, 'The repository id')
```

Comment:
```suggestion
            ->addArgument('repository', InputArgument::REQUIRED, 'The repository ID')
```
?
Or `identifier`
Please edit all occurrences below as well